### ISSUE TITLE
Help please

### ISSUE THREAD

**Issue by jbarros35** [input-output-hk/cardano-metadata-submitter#23]
I don't understand this "base 16" code in your tutorial. Can you please provide us more information? Right now its very hard to follow the instructions.

**Comment by jbarros35** [comment]
To create a new entry, you must first obtain your metadata subject. The subject is defined as the concatenation of the base16-encoded policyId and base16-encoded assetName of your asset. In case your assetName is empty, then the policyId is your subject. We'll consider the following policy for this tutorial:

This part doesn't sound to me explains whats needed to be done

**Comment by gitmachtl** [comment]
you have your policyID (hex-string), thats the **first part** of the subject string.
`example: 34250edd1e9836f5378702fbf9416b709bc140e04f668cc355208518`

if you have also an assetname, you have to convert your ascii assetname into a hex string that will be the **second part** of your subject string, like:
https://github.com/gitmachtl/scripts/blob/a80321bd305dc84bb9b013d8ccc917c968a09298/cardano/testnet/00_common.sh#L467-L472

`example for the assetname "ATADA-Coin": 4154414441636f696e`

then you **combine the two strings together**, this will form your subject.
`example: 34250edd1e9836f5378702fbf9416b709bc140e04f668cc3552085184154414441636f696e`

you can also take a look at my scripts 10-12 in this repo to check the workflow out:
https://github.com/gitmachtl/scripts/tree/master/cardano/testnet
